frontend/packages/integration-tests-cypress/tests/app/poll-console-updates.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { checkErrors } from '../../support';
+import { warningToast } from '../../views/form';
+import { guidedTour } from '../../views/guided-tour';
+
+const CHECK_UPDATES_URL = '/api/check-updates';
+const CHECK_UPDATES_ALIAS = 'checkUpdates';
+const CHECK_MANIFEST_ALIAS = 'checkManifest';
+/* The update wait is the value to wait for the poll of /api/check-updates to return with the updated list of plugins
+ after the plugin is enabled and loaded. This wait will be longer on ci than when debugging locally. */
```

Comment:
I'm not sure the second sentence about taking longer in CI is accurate for these tests.

Reply:
Good point.  Omitted the comment entirely.